logstash-core/src/main/java/org/logstash/config/ir/graph/Graph.java

```diff
@@ -330,14 +330,7 @@ public void validate() throws InvalidIRException {
     }
 
     public String toString() {
-        Stream<Edge> edgesToFormat;
-        try {
-            edgesToFormat = sortedEdges();
-        } catch (InvalidIRException e) {
-            // Even if this isn't a valid graph we still need to print it
-            edgesToFormat = edges.stream();
```

Comment:
This looks like a functional change (not just clean up).  Is this passive?

Reply:
@jakelandis yea this is dead code, the exception is never thrown (the compiler now complains here after removing the redundant exceptions) :)